assets/settings/default.json

```diff
@@ -89,6 +89,12 @@
   // Whether to automatically type closing characters for you. For example,
   // when you type (, Zed will automatically add a closing ) at the correct position.
   "use_autoclose": true,
+  // Controls how the editor handles the autoclosed characters.
+  // When set to `false`(default), skipping over and auto-removing of the closing characters
+  // happen only for auto-inserted characters.
+  // Otherwise(when `true`), the closing characters are always skipped over and auto-removed
+  // no matter how they were inserted.
+  "always_handle_autoclosed_character": false,
```

Comment:
I know it's not the best name either, but `treat_all_brackets_as_auto_closed` would explain things better for me. What do you think? Or `always_treat_brackets_as_autoclosed`?

Reply:
I think `brackets` is also the word we use in other language configurations, even though not all of them are brackets per se. So for consistency's case, I think it makes sense to use `brackets` :)

Reply:
Great, I will rename the flag to `always_treat_brackets_as_autoclosed` then